resqpy/well/_blocked_well.py

```diff
@@ -3154,3 +3154,58 @@ def write_hdf5(self, file_name = None, mode = 'a', create_for_trajectory_if_need
         h5_reg.register_dataset(self.uuid, 'LocalFacePairPerCellIndices', raw_face_indices)  # could use uint8?
 
         h5_reg.write(file = file_name, mode = mode)
+
+    def add_grid_property_to_blocked_well(self, uuid_list):
+        """Add properties to blocked wells from a list of uuids for properties on the supporting grid."""
+
+        part_list = [self.model.part_for_uuid(uuid) for uuid in uuid_list]
+
+        assert len(self.grid_list) == 1, "Only blocked wells with a single grid can be handled currently"
+        grid = self.grid_list[0]
+        parts = self.model.parts_list_filtered_by_supporting_uuid(part_list,
+                                                                  grid.uuid)  # only those properties on the grid
```

Comment:
could be good to warn if len(parts) < len(part_list), as it means one or more of the uuids was not for a property on the grid